Dockerfile

```diff
@@ -1,7 +1,5 @@
-FROM registry.opensource.zalan.do/stups/alpine:latest
-MAINTAINER Team Teapot @ Zalando SE <team-teapot@zalando.de>
+FROM scratch
```

Comment:
Why this change?
It doesn’t help in your issue and it’s not possible to investigate from inside containers anymore.

Reply:
I'll revert this once we agree on final solution, cherry-picked and squashed in rush